src/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { store } from './app/store';
+import App from './App';
+import './index.css';
+import { BrowserRouter } from 'react-router-dom';
+
+const container = document.getElementById('root')!;
+const root = createRoot(container);
+
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
```

Comment:
Maybe this would be interesting to read 
https://reactrouter.com/en/main/start/overview
and 
https://reactrouter.com/en/main/routers/create-browser-router ,
(if you don't read before, of course). This is small enchance from ReactRouter.

Reply:
Thank you for the useful reminder, I read about this feature, but never tried it.